installer_scripts/install-yb-voyager

```diff
@@ -6,19 +6,37 @@ export LC_ALL=en_US.UTF-8
 
 set -e
 
-VERSION="latest"
-if [ "$1" != "" ]
-then
-	if [ "$1" == "local" ]
-	then
-		# Install from locally checked out branch.
-		VERSION=local
-	else
-		echo "ERROR: Invalid argument: '$1'"
-		exit 1
-	fi
+OPTS=$(getopt -o v: --long version:,only-pg-support -n 'install-yb-voyager' -- "$@")
+
+if [ $? != 0 ]; then 
+	echo "Unable to process the options. Exiting..." >&2; 
```

Comment:
Does the user get the exact error why the script couldn't process options?

Reply:
The user gets an error if a wrong option is passed. I think this statement is unnecessary. I found it in a template on stackoverflow. I have removed it for now.
Also a small finding. getopt doesn't match the whole string. For example, the result would be the same if either `--only-pg-support` or `--only `is passed.